src/js/common/schemaform/SchemaForm.jsx

```diff
@@ -92,17 +92,18 @@ class SchemaForm extends React.Component {
     return transformErrors(errors, this.props.uiSchema);
   }
 
-  validate(formData, errors) {
+  validate(pageData, formData, errors) {
     const { schema, uiSchema } = this.props;
     if (uiSchema) {
-      uiSchemaValidate(errors, uiSchema, schema, formData);
+      uiSchemaValidate(errors, uiSchema, schema, pageData, formData);
     }
     return errors;
   }
 
   render() {
     const {
-      data,
+      pageData, // Unfortunately, Form calls this formData, but don't be confused,
```

Comment:
Wait, I'm not seeing where that's true--what exactly are you referring to as Form here?